src/digest.c

```diff
@@ -103,32 +103,52 @@ SEXP is_little_endian(void) {
 }
 
 /* for any algorithm that has output as unsigned char*, can use _store_from_char_ptr */
-#define _store_from_char_ptr(WHAT) if (leaveRaw) {                          \
-    memcpy(output, (WHAT), output_length);                     \
-} else for (int j = 0; j < output_length; j++) {               \
-    snprintf(output + j * 2, 3, "%02x", (WHAT)[j]);            \
-}                                                              \
-
-#define decl_tmp32(WHAT) uint32_t tmp = (WHAT);
-#define decl_tmp64(WHAT) uint64_t tmp = (WHAT);
+void _store_from_char_ptr(
+    const unsigned char *hash, unsigned char *output,
+    const size_t output_length, const int leaveRaw
+) {
```

Comment:
Could I plead with you to have indentation follow the rest of the file, please?  I.e.

```c++
void _store_from_char_ptr(const unsigned char *hash, unsigned char *output,
                          const size_t output_length, const int leaveRaw) {
     // ...

Reply:
absolutely - my bad, should have caught that was the style choice.

Reply:
Thanks. It is the aspect of the 'now common with the kids these days' default choice I like the least as it "blends" function arguments with the function-local variableas as everything is indented the same way.  I find that less readable / imposing extra cognitive load.  Ah well.